src/test/java/http_server/MockRouter.java

```diff
@@ -0,0 +1,33 @@
+package http_server;
+
+public class MockRouter {
+    Router router;
+
+    public MockRouter() {
+        this.router = new Router();
+    }
+
+    public Router getRouter() {
+        initializeRoutes();
+        System.err.println(router);
+        return router;
```

Comment:
What happens if you call this multiple times on the same instance?  I'm guessing since it's just object lookups, it harmlessly overwrites the previous route map?

Reply:
Yup -- it will harmlessly overwrite the previous route map